lib/cli/commands-schema/aws-service.js

```diff
@@ -0,0 +1,271 @@
+'use strict';
+
+const awsServiceOptions = require('./common-options/service');
+
+const commands = (module.exports = new Map());
+
+commands.set('deploy', {
```

Comment:
Should it be only available for `aws-service`? What about support for e.g. https://github.com/serverless/serverless-azure-functions ?